personio-personnel-data-api-oa3.yaml

```diff
@@ -2216,6 +2216,128 @@ paths:
                           "The file must be a file of type: pdf, docx, doc, jpg, zip, png, txt, jpeg, odt, xlsx, rtf, htm, xls, p7s, pptx, pages, rar, ppt, gif, tif, html, msg, asc, tiff, wps, bmp, 7z, csv, ics, vcf, dotx, numbers, eps, gz, ods, otf, odp, odg, rtfd, txz, webarchive, xml, xps, md, fax."
                         ]
 
+  /company/custom-reports/reports:
+    get:
+      tags:
+        - Custom Reports
+      parameters:
+        - name: X-Company-Id
+          in: header
+          schema:
+            type: integer
+          required: true
+        - name: report_ids
```

Comment:
[Question]: If a list of `report_ids` is provided and some of them do not exist, what is the expected response in this case?

Reply:
No exception is expected here. If the reports do not exist, they are not being returned.